ui/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
- [ ] Please change `short_name` and `name`

Reply:
Changed.